src/item_pocket.h

```diff
@@ -338,6 +341,8 @@ class item_pocket
         // the items inside the pocket
         std::list<item> contents;
         bool _sealed = false;
+
+        bool allowed = true; // is it possible to put things in this pocket
```

Comment:
This needs addressing, I think it needs serialization and deserialization so corpses don't lose the flag when they are saved and loaded.

Reply:
Done